module5/ex22_once_per_minute/ex22_once_per_minute.py

```diff
@@ -1,3 +1,33 @@
+import time
+from math import trunc
+
+start_time = 0
+is_first_time = True
+
+
+class TooSoonError(Exception):
+    pass
+
+
+def once_per_minute(func):
```

Comment:
This is failing test `once_per_minute_test.py::test_run_twice_in_a_row`, I'll look into it and come with feedback.